test/e2e/core.go

```diff
@@ -17,140 +17,17 @@ limitations under the License.
 package e2e
 
 import (
-	"fmt"
-	"io/ioutil"
-	"net"
-	"strings"
-	"sync"
+	"log"
+	"os"
+	"os/exec"
+	"path"
 )
 
-type command struct {
-	cmd       string
-	component string
-}
-
 func CoreDump(dir string) {
```

Comment:
a) can you add a docstring? I know it wasn't here before, but may as well add one now.
b) why don't we just move this to util.go? no need to have this in a separate file.